components/tel/Core/Src/xbee.c

```diff
@@ -0,0 +1,87 @@
+/**
+ *  @file xbee.c
+ *  @brief Defines functions related to xBee module, including setting up xBee packet overhead, and handling return frames
+ *
+ *  @date 2024/06/16
+ *  @author Evan Owens
+ */
+
+#include "xbee.h"
+#include <math.h>
+
+/**
+ * @brief Adds up all the bytes after length and before checksum in an api packet
+ *
+ * @param api packet: api packet array
+ * @return uint16_t total_bytes in api packet before checksum and after length fields
+*/
+
+uint16_t XBEE_sum_bytes (uint8_t api_packet[], uint16_t packet_length)
+{
+  uint16_t total_bytes = 0;
+  for (int i = 3; i < packet_length; i++){
+		 total_bytes += api_packet[i];
+  }
+  return total_bytes;
+}
+
+
+/**
+ * @brief Sets up overhead for an API Packet
+ *
+ * @param api packet: api packet array
+ * @return api_packet with setup api overhead
+*/
+void XBEE_api_overhead_setup(uint8_t api_packet[])
+{
+  uint8_t bit_address_64[] = {0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00};
```

Comment:
Why is the bit_address_64 all 0's. Also I would make this a symbolic constant.

Reply:
I wasn't sure how to make a symbolic constant of an array. We need to replace the 00s with the bit address of the radio receiver connected to sunlink

Reply:
I see. For the symbolic constant you do them for each individual byte so long as its in a configurable in a header file.